packages/gestalt/src/PageHeaderComponents.js

```diff
@@ -0,0 +1,210 @@
+// @flow strict
+import { useState, useRef, Fragment, type Element, type Node } from 'react';
+import Box from './Box.js';
+import Heading from './Heading.js';
+import Badge from './Badge.js';
+import Tooltip from './Tooltip.js';
+import IconButton from './IconButton.js';
+import Text from './Text.js';
+import Link from './Link.js';
+import Flex from './Flex.js';
+import Mask from './Mask.js';
+import Image from './Image.js';
+import Dropdown from './Dropdown.js';
+import { type ActionType } from './PageHeader.js';
+
+export function PageHeaderTitle({ title }: {| title: string |}): Node {
+  return (
+    <Fragment>
+      <Box display="block" smDisplay="none">
+        <Heading size="400" lineClamp={1} accessibilityLevel={1}>
+          {title}
+        </Heading>
+      </Box>
+      <Box
+        display="none"
+        smDisplay="block"
+        dangerouslySetInlineStyle={{ __style: { marginTop: '-7px' } }}
+      >
+        <Heading size="500" lineClamp={1} accessibilityLevel={1}>
+          {title}
+        </Heading>
+      </Box>
+    </Fragment>
+  );
+}
+
+export function PageHeaderThubnail({ thumbnail }: {| thumbnail?: Element<typeof Image> |}): Node {
```

Comment:
typo: PageHeaderThumbnail (missing 'm')

also: the prop should be required

Reply:
DONE